app/helpers/rad_common/security_role_helper.rb

```diff
@@ -31,5 +31,9 @@ def normalize_name(label)
         label
       end
     end
+
+    def collection
```

Comment:
we should probably name this something since a  helper is in the global scope, `collection` is very general